From 7211f38a828b08b15b003e2d44142d1b8d9d1610 Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Fri, 9 Mar 2018 07:08:09 +0100
Subject: [PATCH 61/74] libfm_jni: Resolve fread assignment to condition cast

 * libfm_jni/ConfFileParser.cpp:480:21:
    error: using the result of an assignment
    as a condition without parentheses

Change-Id: Ib9526734ec96e029f89801ec9b008296a7e2c0fc
---
 libfm_jni/ConfFileParser.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libfm_jni/ConfFileParser.cpp b/libfm_jni/ConfFileParser.cpp
index 5897992..04c9c0d 100644
--- a/libfm_jni/ConfFileParser.cpp
+++ b/libfm_jni/ConfFileParser.cpp
@@ -477,7 +477,7 @@ static char parse_load_frm_fhandler
   unsigned int i;
   bool has_carriage_rtn = false;
 
-  while (bytes_read = fread(buf, 1, MAX_LINE_LEN, fp)) {
+  while ((bytes_read = fread(buf, 1, MAX_LINE_LEN, fp))) {
         for (i = 0; i < bytes_read; i++) {
             if (line_len == line_allocated) {
                 line_allocated += 25;
-- 
2.17.1

